### DISCUSSION TITLE
不知道实验二怎么入手，有没有大佬可以大致说下思路，或者有没有类似其他的参考样例程序学习资料等等

### DISCUSSION THREAD

**Discussion by Gary-wugq** [arcsysu/SYsU-lang#25] [Q&A] [answered]
Help! (!_!)

**Comment by wu-kan** [comment, marked as answer]
在 [这里](https://github.com/arcsysu/SYsU-lang/blob/main/parser/README.md) 更新了一个详细一些的例子，不知道是否有帮助？

**Reply by Aurora-1412** [reply]
请问助教在实验课上演示的那个北航实验的链接在哪里orz，（不是github仓库，是一个gitbook）

**Reply by wu-kan** [reply]
<https://buaa-se-compiling.github.io/miniSysY-tutorial/>
